packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -189,30 +239,46 @@ export const useCardStyles_unstable = (state: CardState): CardState => {
     large: styles.sizeLarge,
   } as const;
 
-  const interactive =
-    state.root.onClick ||
-    state.root.onMouseUp ||
-    state.root.onMouseDown ||
-    state.root.onPointerUp ||
-    state.root.onPointerDown ||
-    state.root.onTouchStart ||
-    state.root.onTouchEnd;
```

Comment:
`selectable` prop now controls intractability.

Reply:
Actually, this was an incorrect assumption. Card should still get interactive styles without `selectable` if it is "interactive". We'll need to double check what interactive means but for now we'll keep this logic.